charts/consul/templates/crd-gatewaypolicies.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Is this intentional? I suspect it's a change due to the version of tooling installed locally

Reply:
yeah it is, there were changes added to main that specify the version of this tool to be `v0.8.0` that were pulled in on a rebase, my version previously was 0.11.3

Reply:
TIL